src/Compilers/CSharp/Portable/BoundTree/BoundBinaryOperator.UncommonData.cs

```diff
@@ -11,11 +11,29 @@ internal partial class BoundBinaryOperator
     {
         internal class UncommonData
         {
+            public static UncommonData UnconvertedInterpolatedStringAddition(ConstantValue? constantValue) =>
+                new UncommonData(
+                    constantValue,
+                    method: null,
+                    constrainedToType: null,
+                    originalUserDefinedOperatorsOpt: default,
+                    isUnconvertedInterpolatedStringAddition: true,
+                    interpolatedStringHandlerData: null);
+
+            public static UncommonData WithInterpolatedStringHandlerData(InterpolatedStringHandlerData data)
```

Comment:
>WithInterpolatedStringHandlerData [](http://example.com/codeflow?start=39&length=33)

There is no instance to "With". Naming should probably align with `UnconvertedInterpolatedStringAddition` above.